src/postman/operation.ts

```diff
@@ -1,3 +1,81 @@
-// import { PostmanCollectionHttpOperationTransformer } from './types';
+import {
+  HttpParamStyles,
+  IHttpHeaderParam,
+  IHttpOperationRequestBody,
+  IHttpParam,
+  IHttpPathParam,
+  IHttpQueryParam,
+  IMediaTypeContent,
+} from '@stoplight/types';
+import { Collection, HeaderList, Item, ItemGroup, Url } from 'postman-collection';
+import { transformBody, transformHeader, transformPathParams, transformQueryParam } from './transformers/request';
+import { PostmanCollectionHttpOperationTransformer } from './types';
+import { transformDescriptionDefinition, traverseItemsAndGroups } from './util';
 
-// export function transformPostmanCollectionOperation({}): PostmanCollectionHttpOperationTransformer {}
+export const transformPostmanCollectionOperation: PostmanCollectionHttpOperationTransformer = ({
+  document,
+  path,
+  method,
+}) => {
+  const collection = new Collection(document);
+  const resolvedCollection = new Collection(collection.toObjectResolved({ variables: collection.variables }, []));
+
+  const item = findItem({ collection: resolvedCollection, method, path });
+  if (!item) {
+    throw new Error(`Unable to find "${method} ${path}"`);
+  }
+
+  const header = item.request.headers.all().map(transformHeader);
+
+  return {
+    id: '?http-operation-id?',
+    iid: item.id,
+    description: item.description && transformDescriptionDefinition(item.description),
+    method,
+    path,
+    summary: item.name,
+    request: {
+      query: item.request.url.query.all().map(transformQueryParam),
+      header,
+      path: transformPathParams(item.request.url.path),
+      body: item.request.body ? transformBody(item.request.body, findContentType(item.request.headers)) : undefined,
+    },
+    responses: [],
+    /*
+    servers: ...,
+    security: ...,
+    */
+  } as any;
```

Comment:
I know

Reply:
Yes, `responses[]` requires at least one response so I needed to cast it and yolo1!!